src/editor/UI/IndentLineRenderer.re

```diff
@@ -11,6 +11,53 @@ open Oni_Model;
 
 type bufferPositionToPixel = (int, int) => (float, float);
 
+let rec _getIndentLevel =
```

Comment:
Optional nit: this isn't specific to this PR just happened to mention this time, but I think we should avoid using `_myPrivateVariable` as a way of hiding variables from being exported.

I think if we feel strongly enough that something shouldn't be exported we should use an interface file (`rei`) especially since the compiler reads `_` as things we aren't using.

Reply:
Good point @Akin909 (I need to be better about including `rei` files). 

I wonder if there is a way to lint / check this in our CI to enforce it? I've made this mistake a few times